legal-api/Makefile

```diff
@@ -131,7 +131,10 @@ tag: push ## tag image
 # COMMANDS - Local                                                              #
 #################################################################################
 run: ## Run the project in local
-	. venv/bin/activate && python -m flask run -p 5000
+	. venv/bin/activate && python -m flask run -p 5050
+
+run-websockets: ## Run the project in local with websockets
+	. venv/bin/activate && python -m gunicorn --threads 100 --bind :5050 wsgi
```

Comment:
How many threads were you thinking of needing for Prod per pod?  Right now, we typically have 3 pods running for the legal api.

Have you used websockets previously with threads for a flask api for digital credential work?  If so, how was the performance?  Any other tweaks required other than the thread count?

Reply:
This would be the first I'm using web sockets with a Flask application. Although I believe a recent project `vc-authn-oidc` was rewritten using Flask and I believe does use web sockets. I think @esune might have some advice on this.

Reply:
Just a heads up that we already have config(`GUNICORN_PROCESSES` & `GUNICORN_THREADS`) for the process and thread count in the legal api for gunicorn.  processes=2 and threads=5 for all environments right now. 

Akiff, let me know if you are thinking we need the thread count bumped.

Reply:
I think we can start with the config that the legal API is using and tweak it if we see performance issues. I am not an expert by any means at sizing these parameters, but don;t expect it would be an issue unless we have LOTS of concurrent users connecting, which seems unlikely at least at the beginning.